db/schema.rb

```diff
@@ -23,6 +23,13 @@
     t.datetime "image_updated_at"
   end
 
+  create_table "admin_admission_eligibilities", force: :cascade do |t|
```

Comment:
Why it is exist in this schema file ? Is the story tell you to load `admin_admission_eligibilities` in schema file ?